recipes/quill/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "quill" %}
+{% set version = "4.3.0" %}
+{% set sha256 = "c97bf3bfac6dfb7ed77fa08d945a490e302ba07e405539fda61985b39750cb29" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/odygrd/quill/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

No need to template single use variables. That obscures the recipe. We prefer to keep the jinja at a minimum, just enough to help the auto-updating bot to run.

```suggestion

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://github.com/odygrd/quill/archive/refs/tags/v{{ version }}.tar.gz
  sha256: c97bf3bfac6dfb7ed77fa08d945a490e302ba07e405539fda61985b39750cb29
```